src/types.ts

```diff
@@ -27,8 +27,11 @@ export interface PromClient {
     increaseTotalBlocksProduced(name: string, address: string): void;
     increaseTotalValidatorOfflineReports(name: string, address: string): void;
     resetTotalValidatorOfflineReports(name: string): void;
-    setStateValidatorOfflineSessionReports(name: string): void;
-    resetStateValidatorOfflineSessionReports(name: string): void;
+    setStatusValidatorOffline(name: string): void;
```

Comment:
An interface with so many methods is screaming for a refactor :) but not as part of this PR of course, we can work on it later.